### PR TITLE
Replace old pastebin link that was deleted

### PR DESCRIPTION
**Pull request by ArdentLeKhey**
• Replace old pastebin link that was deleted
• Improve documentation with example and event linked to the pickup item

### CODE REVIEW

File: OBJECT/CreateAmbientPickup.md [hunk 1]
```diff
@@ -25,3 +26,8 @@ Pickup hashes: pastebin.com/8EuSv2r1
 * **p8**: 
 
 ## Return value
+
```
Comment (4mmonium) [inline OBJECT/CreateAmbientPickup.md:29]: Missing `## Examples` header.  I covered this on this [PR](https://github.com/citizenfx/natives/pull/838/files). The native also seems to do a check for cash pickups. I will research it more thoroughly soon, thanks for the contribution though! :-)

Comment (ArdentLeKhey) [reply OBJECT/CreateAmbientPickup.md:29]: ok nice,
is there a boilerplate for native documentation ? (so I can do better next time)
As for the check for cash pickup I don't know what the game does exactly, but whatever happens the pickup pickup and disappearance of the pickup is done automatically when the player passes over it (even for cash pickups) and the event CEventNetworkPlayerCollectedAmbientPickup is trigger.
As for the native flags, I remember seeing a pastebin on them years ago but I can't find it anymore and it's a shame.

Comment (4mmonium) [reply OBJECT/CreateAmbientPickup.md:29]: > ok nice, is there a boilerplate for native documentation ?

Yes, you can find the contribution guide [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md)
As for native flags, I was going to do research on that.

File: OBJECT/CreateAmbientPickup.md [hunk 2]
```diff
@@ -25,3 +26,8 @@ Pickup hashes: pastebin.com/8EuSv2r1
 * **p8**:
```
Comment (4mmonium) [inline OBJECT/CreateAmbientPickup.md:26]: Take a look at the code I posted, I was able to figure out `p8` is `netMissionEntity` by looking at the native code for [`CREATE_VEHICLE`](https://docs.fivem.net/natives/?_0xDD75460A) in the game's binary.

**Comment by thorium-cfx** [comment]
Closing this as it became stale

**Pull request closed without merging**